roles/unbind-unifiedpush-apb/tasks/main.yml

```diff
@@ -1,2 +1,12 @@
+# Remove annotation from mobile client 
+- set_fact:
+    upsUrlAnnotation: "org.aerogear.binding.{{ _apb_bind_creds.upsInstanceName }}/ups_url"
+
+- name: Remove annotations from client {{ _apb_bind_creds.clientId }}
+  shell: "oc annotate mobileclient {{ _apb_bind_creds.clientId }} {{ item }}- -n {{ namespace }}"
+  ignore_errors: yes
+  with_items:
+    - "{{ upsUrlAnnotation }}"
+
 - debug:
-    msg: "Currently unused (handled by ups-config-operator)"
\ No newline at end of file
+    msg: "The rest of unbind steps is handled by ups-config-operator"
```

Comment:
I meant to ask @pb82 why the APB did not implement the unbind step?

Reply:
There's no need for an unbind role. When you delete the binding, the operator will be triggered and take care of everything.